superbench/benchmarks/micro_benchmarks/__init__.py

```diff
@@ -10,8 +10,9 @@
 from superbench.benchmarks.micro_benchmarks.cublas_function import CublasBenchmark
 from superbench.benchmarks.micro_benchmarks.cudnn_function import CudnnBenchmark
 from superbench.benchmarks.micro_benchmarks.gemm_flops_performance import GemmFlopsCuda
+from superbench.benchmarks.micro_benchmarks.cuda_memory_bw_performance import CudaMemBwBenchmark
 
 __all__ = [
     'MicroBenchmark', 'MicroBenchmarkWithInvoke', 'ShardingMatmul', 'ComputationCommunicationOverlap', 'KernelLaunch',
-    'CublasBenchmark', 'CudnnBenchmark', 'GemmFlopsCuda'
+    'CublasBenchmark', 'CudnnBenchmark', 'GemmFlopsCuda', 'CudaMemBwBenchmark'
```

Comment:
'CudaMemBwBenchmark' => 'MemBwCuda'? Just like GemmFlopsCuda.

Reply:
you have a conflict with @cp5555 
'CudnnBenchmark' and 'GemmFlopsCuda' are already different styles, I would suggest to include original name

Reply:
@cp5555 @guoshzhao @abuccts  please make a final decision and tell me

Reply:
Yes, I think we can keep `CudaXXXBenchmark` for this. For `GemmFlopsCuda`, we should change to `CudaGemmFlopsBenchmark`.

Reply:
@abuccts Weird, when I reviewed the PR, I just saw two comments from @cp5555, and no comments related with naming. 

Any choice are OK to me as long as we used unified naming format.